doc/parameters/param_list.md

```diff
@@ -175,7 +175,19 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | Parameter | Default | Description |
 | ----------| --------| ------------|
 | `<filter name>`.enabled | true | Whether it is enabled |
-| `<filter name>`.filter_info_topic | N/A | Name of the CostmapFilterInfo topic having filter-related information |
+| `<filter name>`.filter_info_topic | N/A | Name of the intput CostmapFilterInfo topic having filter-related information |
+| `<filter name>`.transform_tolerance | 0.0 | TF tolerance |
+
+## speed filter
+
+* `<filter name>`: Name corresponding to the `nav2_costmap_2d::SpeedFilter` plugin. This name gets defined in `plugins`.
+
+| Parameter | Default | Description |
+| ----------| --------| ------------|
+| `<filter name>`.enabled | true | Whether it is enabled |
+| `<filter name>`.filter_info_topic | N/A | Name of the input CostmapFilterInfo topic having filter-related information |
+| `<filter name>`.speed_limit_topic | N/A | Name of the output speed limiting topic |
```

Comment:
I think there should be a default speed limit topic the same as the default for DWB, that way it works out of the box even without configuration

Reply:
Done in https://github.com/ros-planning/navigation2/pull/2074/files#diff-db47883307b4ff824eaf471c4e80023d361fb3f9d78a32731288e896c8063b6bR189